assets/base/i18n/en.json

```diff
@@ -555,7 +555,6 @@
   "notification_settings.auto_responder.default_message": "Hello, I am out of office and unable to respond to messages.",
   "notification_settings.auto_responder.enabled": "Enabled",
   "notification_settings.auto_responder.footer_message": "Set a custom message that will be automatically sent in response to Direct Messages. Mentions in Public and Private Channels will not trigger the automated reply. Enabling Automatic Replies sets your status to Out of Office and disables email and push notifications.",
-  "notification_settings.auto_responder.message_placeholder": "Message",
```

Comment:
Was removing this intentional? Not sure if it's related to this PR.

Reply:
Script remove it automatically if ko longer present in the codebase